api/v1alpha1/oadp_types.go

```diff
@@ -218,6 +218,12 @@ type DataMover struct {
 	// defines the parameters that can be specified for retention of datamover snapshots
 	// +optional
 	SnapshotRetainPolicy *RetainPolicy `json:"snapshotRetainPolicy,omitempty"`
+	// schedule is a cronspec (https://en.wikipedia.org/wiki/Cron#Overview) that
+	// can be used to schedule replication to occur at regular, time-based
+	// intervals.
+	//+kubebuilder:validation:Pattern=`^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$`
+	//+optional
+	Schedule string `json:"schedule,omitempty"`
```

Comment:
How would one setting this setting affect backup or restore with data mover?

I would add description to provide a one usecase example so they can see via `oc explain` and other mechanisms (openshift web console)

Reply:
This is part of the [requirement in the jira](https://issues.redhat.com//browse/OADP-2048) as well

> Instructions for setup can be found here: `oc explain dpa.Spec.Features.DataMover.Schedule`

Reply:
at minimum could link to https://volsync.readthedocs.io/en/stable/usage/triggers.html is my 2cents.

Reply:
Triggers are used for starting the Volsync synchronization process, by default we had manual triggers, this PR helps add a schedule trigger.

Reply:
Done, updated the PR.